scripts/create_adhoc_redis_service.sql

```diff
@@ -0,0 +1,73 @@
+-- Add the missing historic recorded service in prod
```

Comment:
Why only in prod? I don't like adding state manually, and I don't like that it won't be in dev or staging. I think this should be codified.

Reply:
If I add it to the initialisation of the app, we need to change all the tests to reflect that new thing.

That is why I did not codified automatically. 

This is to add the data once on production. The `service_plans` data is populated every now and then with data from CF and it is not added programaticaly.

Reply:
> If I add it to the initialisation of the app, we need to change all the tests to reflect that new thing.

I don't think that justifies having to manually run a script to make prod work, especially with the new added complexity of multiple prod environments. This app is complicated enough to understand without the added complexity of remembering ad hoc things we layered on top of it.

I think we should codify this.

Reply:
But I insist, this is one off ever

Reply:
@keymon discussed this offline and agreed scripting against prod is okay because:

- the data is only needed in prod (specifically the existing Ireland environment). This is because of event data for a service plan exists when the service plan has been removed _before_ we started collecting historical service plan data.
- there is extra work in correcting existing tests if it is added everywhere
- this only needs to be run once